src/worknet/Commands/ContractCommand.Storage.List.cs

```diff
@@ -0,0 +1,62 @@
+using System.ComponentModel.DataAnnotations;
+using System.IO.Abstractions;
+using McMaster.Extensions.CommandLineUtils;
+using Neo;
+using Neo.BlockchainToolkit.Models;
+using NeoWorkNet.Node;
+
+namespace NeoWorkNet.Commands
+{
+  partial class ContractCommand
+  {
+    // [Command("storage", Description = "Manage neo-worknet values")]
+    // [Subcommand(typeof(Update))]
+    internal partial class Storage
+    {
+      [Command("list", Description = "List all storage values associated with a contract")]
+      internal class List
+      {
+        readonly IFileSystem fs;
+        public List(IFileSystem fileSystem)
+        {
+          this.fs = fileSystem;
+        }
+
+        [Argument(0, Description = "Contract name or hash")]
+        [Required]
+        internal string Contract { get; init; } = string.Empty;
```

Comment:
can we add a `--json` option to emit the results as JSON

Reply:
this is a sample output: {"script-hash":"0x50695ace7ab6c5f42d45828d36281bebd83201ee","storages":[{"key":"0xFF","value":"0xABCD"}]}